assets/training/aoai/proxy_components/src/finetune_submit.py

```diff
@@ -40,27 +46,72 @@ def submit_finetune_job(self, training_file_id, validation_file_id, model, regis
             validation_file=validation_file_id,
             hyperparameters=hyperparameters,
             suffix=suffix)
-        job_id = finetune_job.id
+        self.job_id = finetune_job.id
         status = finetune_job.status
 
-        # If the job isn't yet done, poll it every 30 seconds.
-        if status not in ["succeeded", "failed"]:
+        terminal_statuses = ["succeeded", "failed", "cancelled"]
+        # If the job isn't yet done, poll it every 60 seconds.
+        if status not in terminal_statuses:
```

Comment:
can we use specific status, like running? else we might end up doing following steps for status we dont intend to

Reply:
status of finetuning job can be one of the following: 
terminal statuses : "succeeded", "failed", "cancelled"
non-terminal statuses: "validating_files", "queued", "running"
until status is not terminal, we will poll status